src/components/Member.svelte

```diff
@@ -0,0 +1,32 @@
+<script>
+  export let member = {};
+  let { imageSrc, name, bio, socials = [], website = {} } = member;
+</script>
+
+<div class="max-w-sm rounded overflow-hidden shadow-xl">
+  <div class="h-64 flex-none bg-center bg-cover rounded-t lg:rounded-t-none lg:rounded-l text-center overflow-hidden" style="background-image: url('/members/{imageSrc}')"/>
+  <div class="px-6 py-4">
+    <div class="font-bold text-xl mb-2">
+      {name}
+    </div>
+    <p class="text-gray-700 text-base">
+      {bio}
+    </p>
+  </div>
+  <div class="px-6 py-4 flex justify-between">
+    {#if socials}
```

Comment:
socials is an array. We should check for `#if socials.length > 0` here

Reply:
I'm always wondering if I should write `array.length` oder `array.length > 0 `. Snce the first option is 0 or nullable

Reply:
(but fixed :) )